src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+import java.io.Serializable;
+
+public class Task implements Serializable{
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public void markAsDone(){
+        this.isDone = true;
+    }
+
+    public void markAsNotDone(){
```

Comment:
I realize you have done this in several other code files, so I guess you could just follow that.